### DISCUSSION TITLE
Allow toolbar config through django settings

### DISCUSSION THREAD

**Discussion by buswedg** [withlogicco/django-prose#96] [Ideas]
Considered adding a config wrapper, such that the toolbar buttons can be configured via Django settings?

Django-code-mirror has an implementation like this which is quite handy as an example: https://github.com/pavelsof/django-mirror/blob/master/django_mirror/widgets.py

I'm just starting to get my head around trix. But I see some basic customizations noted here: https://matthaliski.com/blog/customizing-the-trix-toolbar

**Comment by parisk** [comment]
This looks a bit similar to https://github.com/withlogicco/django-prose/discussions/31 and IMO is a good idea.

Would make sense to include documentation for users on how to create their own widget template, CSS and JavaScript to do so? This could be accompanied by the following settings


```py
DJANGO_PROSE = {
  EDITOR_WIDGET_TEMPLATE = ...
  EDITOR_WIDGET_JS = ...
  EDITOR_WIDGET_CSS = ...
}
```
